modules/react/tabs/lib/TabsItem.tsx

```diff
@@ -126,7 +127,9 @@ export const StyledTabItem = styled(Box.as('button'))<StyledType & Pick<TabsItem
       [systemIconStencil.vars.color]: theme.canvas.palette.primary.main,
       '&:after': {
         position: 'absolute',
-        height: space.xxxs,
+        // height: space.xxxs,
```

Comment:
```suggestion
        height: space.xxxs,
```
I believe this was done for:
<img width="177" alt="image" src="https://github.com/user-attachments/assets/d75874c0-df97-478c-bf3e-bb3139019bbc" />
 but showing the selected border bottom should be fine.